src/components/RelatedVideos/RelatedVideos.component.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import VideoCard from '../VideoCard';
+
+function RelatedVideos({ relatedVideos }) {
+  return (
+    <div className="related-videos-wrapper">
```

Comment:
I think that you can use a [Fragment](https://reactjs.org/docs/fragments.html) here.